src/main/java/mate/academy/shop/anotation/Injector.java

```diff
@@ -9,11 +12,13 @@
 import java.util.List;
 
 public class Injector {
+    final static Logger logger = Logger.getLogger(FileReader.class);
```

Comment:
```suggestion
    private final static Logger logger = Logger.getLogger(Injector.class);
```
the same in with all loggers

Reply:
changed for all